server/views/pages/error.njk

```diff
@@ -1,12 +1,31 @@
 {% extends "../partials/layout.njk" %}
 
-{% set pageTitle = applicationName + " - Error" %}
+{% block pageTitle %}
+  {{ locale.page.title }}
+{% endblock %}
+
 {% set mainClasses = "app-container govuk-body" %}
 
-{% block content %}
+{% block header %}
+  {% include "../partials//header.njk" %}
+{% endblock %}
 
-  <h1>{{ message }}</h1>
-  <h2>{{ status }}</h2>
-  <pre>{{ stack }}</pre>
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">{{ locale.mainHeading.title }}</h1>
+      {% for suggestion in locale.suggestions %}
+        <p class="govuk-body">{{ suggestion }}</p>
+      {% endfor %}
+      {{ locale.resolution | safe }}
+    </div>
+  </div>
 
+  <div class="govuk-grid-row govuk-!-margin-top-4">
+    <div class="govuk-grid-column-full">
+      <code class="code-block">
+        <pre>{{ stack }}</pre>
+      </code>
+    </div>
+  </div>
```

Comment:
If stack is `null` then this should not be included?

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/227/commits/c833e7674387179e5b647d82cdbd785fd0a901ce